index.fa.md

```diff
@@ -0,0 +1,2474 @@
+---
+title: نامه سرگشاده ای برای برکناری ریچارد ام. استالمن از همه مناصب رهبری
+description: ما ، امضا کنندگان ، خواستار استعفا کل هیئت FSF و برکناری ریچارد ام. استالمن از تمام مناصب رهبری هستیم. ما دیگر به رنج رفتارهای وی ، دادن نقش رهبری به او یا ادامه پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ادامه نخواهیم داد. 
+image: /assets/og-image.png
+twitter:
+  card: summary_large_image
+share_text: من از این فراخوان برای کنار رفتن کل هیئت FSF و برکناری ریچارد ام. استالمن از سمت های رهبری حمایت می کنم. 
+---
+
+{% include share-links.html %}
+
+_2021-03-23_
+
+ریچارد ام. استالمن ، که اغلب با نام RMS شناخته می شود ، برای مدت طولانی یک نیروی خطرناک در جامعه نرم افزار آزاد بوده است. او خود را زن ستیز ، توانمند و ترنسفوبیک نشان داده است ، از جمله دیگر اتهامات جدی به بی ادب بودن است. این نوع باورها جایی در نرم افزار رایگان ، حقوق دیجیتال و جوامع فناوری ندارند. با اعاده مجدد وی به عضو هیئت مدیره بنیاد نرم افزار آزاد ، ما خواهان کنار رفتن کل هیئت FSF و برکناری RMS از کلیه مناصب رهبری هستیم.
+
+ما ، امضا شدگان ، به ضرورت خودمختاری دیجیتال و نقش پرقدرت آزادی کاربر در حمایت از حقوق بنیادی انسانی خود ایمان داریم. برای تحقق وعده هر آنچه آزادی نرم افزار امکان پذیر است ، باید تغییرات اساسی در جامعه ایجاد شود. ما به حال و آینده ای اعتقاد داریم که فن آوری به مردم توانمندی  (و نه ظلم) می دهد. ما می دانیم که این تنها در جهانی امکان پذیر است که فناوری برای احترام به حقوق ما در اساسی ترین سطح خود ساخته شود. در حالی که این ایده ها توسط ریچارد ام. استالمن به نوعی رواج یافته است ، اما او به جای ما صحبت نمی کند. ما از اعمال و نظرات وی چشم پوشی نمی کنیم. ما رهبری وی یا رهبری بنیاد نرم افزار آزاد را که امروز برقرار است ، تأیید نمی کنیم. 
+
+تا کنون تحمل کافی عقاید و رفتارهای نفرت انگیز RMS وجود داشته است. ما نمی توانیم ادامه دهیم که یک نفر معنای کار ما را خراب کند. جوامع ما فضایی برای افرادی مانند ریچارد ام. استالمن ندارند و ما با رفتارهای وی ، دادن نقش رهبری به او و یا در غیر اینصورت پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ، ادامه نخواهیم داد. 
+
+**ما خواهان حذف کل اعضای بنیاد نرم افزار آزاد هستیم.** اینها افرادی هستند که سالها RMS را فعال و توانمند کرده اند. آنها با اجازه دادن به او برای عضویت مجدد در شورای FSF این امر را دوباره نشان می دهند. وقت آن است که RMS از نرم افزار آزاد ، اخلاق فناوری ، حقوق دیجیتال و جوامع فناوری عقب نشینی کند ، زیرا او نمی تواند رهبری مورد نیاز ما را تأمین کند. **ما همچنین خواهان برکناری ریچارد ام. استالمن از کلیه مناصب رهبری ، از جمله پروژه GNU هستیم.**
```

Comment:
For the first line: there does not appear to be a reference to the board of the Free Software Foundation. (I'm relying on machine translation). We want to remove all members of the FSF Board.

Reply:
issue has been fixed now.